plugin/trino-hive/src/test/java/io/trino/plugin/hive/AbstractTestHiveFailureRecovery.java

```diff
@@ -70,7 +70,7 @@ public void testRefreshMaterializedView()
                 .hasMessageContaining("This connector does not support creating materialized views");
     }
 
-    @Test(invocationCount = INVOCATION_COUNT)
+    @Test
```

Comment:
> Run failure recover tests once

i don't understand why. Would you mind capturing reasoning in the cmt msg?

also, should this be squashed into the commit which added `INVOCATION_COUNT` in the first place?

Reply:
I increase invocation count to minimize a chance of false negative. Since now we are running the same set of tests twice (once for S3 and once for local) i think it is fine to run only once. But for local testing it is nice to have a single place to increase number of runs (and leave it running for a couple of hours to catch potential memory leaks / false negatives). Could you please simply decrease the value to 1 instead of removing the constant?

Reply:
I left the constant. Will add comment in the commit message